### DISCUSSION TITLE
Does anyone find it ironic that to submit a complaint you have to include your phone number, but the payers don't have contact info?

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#727] [Q&A]
Does anyone find it ironic that to submit a complaint about a TiC file you have to include your phone number and email address, but the files themselves nor the websites hosting them are required to have have contact info for the payer?

Why isn't it a requirement that every TiC file includes set of fields with contact information for the producer/maintainer of the file? It would be a small addition to the schema and it would promote accountability and make it much easier to give feedback to payers.

**Comment by bryanbriegel** [comment]
I agree with the thoughts however this would most likely require rule making, updated ICR as well as burden projections if this were to be required.

**Comment by magician25** [comment]
I am not sure if this is useful but for ACA PUF files, they disclosed the email of tech POC for the issuers. Possibly the same team may be handling the TiC files as well. Please find the file attached.
[machine_readable_PUF (3).xlsx](https://github.com/CMSgov/price-transparency-guide/files/13587916/machine_readable_PUF.3.xlsx)

**Reply by nmayle** [reply]
Thank you.  That is very helpful for the payers who offer exchange plans.  I had missed this source. 
This will be a much better starting point than the random customer service people one reaches via the public email addresses or phone numbers. Even if the contact person for the PUF files isn't involved in TiC, I would imagine they are likely to know where in the organization they live.

Yesterday, I tried contacting a payer via the email address the payer had posted on their TiC MRF page, but the address bounced.
